test/testUtils.js

```diff
@@ -105,3 +105,11 @@ function resetSettings(settingState) {
     bkg.settings.updateSetting('trackerBlockingEnabled', settingState)
     bkg.settings.updateSetting('httpsEverywhereEnabled', settingState)
 }
+
+function clearCache () {
+    return new Promise((resolve) => {
+        const millisecondsPerWeek = 1000 * 60 * 60 * 24 * 7
+        const oneWeek = (new Date()).getTime() - millisecondsPerWeek
+        chrome.browsingData.remove({'since': oneWeek}, {'cache': true}, () => resolve())
```

Comment:
@jdorweiler Question for you here: Chrome api docs say that the extension manifest needs to ask permission to use `chrome.browsingData`, but since we're only using it in test layer I'm guessing (hoping) that does not apply here.

Reply:
That's fine. You just need to remember to add it to the manifest for testing.